src/components/Modals/SpendingMotionModal.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import PropTypes from 'prop-types';
+
+import { useDispatch, useSelector } from 'react-redux';
+import ModalRoot from './ModalRoot';
+import Button from '../Button/Button';
+import { CheckboxInput, SelectInput, TextInput } from '../InputComponents';
+import { congressActions } from '../../redux/actions';
+
+import styles from './styles.module.scss';
+
+function SpendingMotionModal({ closeModal }) {
+  const dispatch = useDispatch();
+
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+  } = useForm({
+    mode: 'all',
+    defaultValues: {
+      spending: 'sendToPolitipool',
+    },
+  });
+
+  const onSubmit = ({ transferToAddress, transferAmount, spending }) => {
+    dispatch(congressActions.createLlmSpendingMotion.call({ transferToAddress, transferAmount, spending }));
+    closeModal();
+  };
+
+  return (
+    <form
+      className={styles.getCitizenshipModal}
+      onSubmit={handleSubmit(onSubmit)}
+    >
+      <h3 className={styles.h3}>Create new spending</h3>
+      <span className={styles.description}>
+        Here you can create a new motion for LLM spending.
+      </span>
+
+      <div className={styles.title}>Liquidity</div>
+      <SelectInput
+        register={register}
+        name="spending"
+        options={[
+          { value: 'sendToAddress', display: 'Send LLM to address' },
+          { value: 'sendToPolitipool', display: 'Send LLM to politipool' },
```

Comment:
That's confusing. We always send to address, it's just that we either send it as liquid or politipooled.